notify/notify.yaml

```diff
@@ -222,6 +242,15 @@ components:
         token_id:
           type: string
           description: token ID for NFT. Can be integer or hex (0x prefixed hex string).
+    nft_metadata_filter_object:
+      type: object
+      properties:
+        contract_address:
+          type: string
+          description: Contract address for an NFT. If this field and the token_id aren't set all metadata updates will be sent.
```

Comment:
Minor point but is there a way to mark `token_id` as a variable here

Reply:
I took a brief look and couldn't find a way :'(